pymatgen/electronic_structure/boltztrap2.py

```diff
@@ -56,16 +56,162 @@
 __date__ = "August 2018"
 
 
-class BandstructureLoader:
-    """Loader for Bandsstrcture object"""
+class PMGLoader:
```

Comment:
This name is too uninformative. it can literally mean loading anything in pymatgen.

Reply:
what about VasprunBSLoader ;-)